apps/librelingo_types/librelingo_types/data_types.py

```diff
@@ -197,7 +200,7 @@ class Module(
     ### Usage examples:
 
     ```python
-    my_module = Module(title="Basics", skills=[skill1, skill2])
+    my_module = Module(title="Basics", filenama="basic/module.yaml", skills=[skill1, skill2])
```

Comment:
```suggestion
    my_module = Module(title="Basics", filename="basic/module.yaml", skills=[skill1, skill2])
```

Reply:
ah, so how does it work then? why are the tests passing?

Reply:
Oh, I see you made the typo not I. Wow I was confused here

Reply:
yes the typo is just in the string, it's not actual code that's compiled. it would be nice to have some linting tool that can notice these mistakes even in documentation. 

some time ago I was experimenting with the idea of forcing code in documentatino to compile. Not sure if there's a way to do that in python :thinking: 

perhaps doctest would be that actually https://docs.python.org/3/library/doctest.html

but anyways that goes beyond the scope of this PR